README.md

```diff
@@ -118,34 +118,32 @@ or you can copy mage folder inside **MAGE** docker container and just do the reb
 Jump to [build MAGE query modules with Docker](#building-and-loading-modules-inside-memgraph-with-docker)
 
 
-### 2. Installing MAGE locally with Linux package of Memgraph
+### 2. Installing MAGE on Linux distro from source
 > Note: This step is more suitable for local development.
 
 #### Prerequisites
 * Linux based Memgraph package you can download [here](https://memgraph.com/download). We offer Ubuntu, Debian, Centos based Memgraph
 packages. To install, proceed to the following [site](https://memgraph.com/docs/memgraph/installation).
 * To build and install MAGE query modules you will need: **Python3**, **Make**, **CMake**, **Clang**, **UUID** and **Rust**.
 
-
 Since Memgraph needs to load MAGE's modules, there is the `setup` script to help you.
 
-By running following command, this script will change default directory where Memgraph is looking for query modules to your
-`mage/dist` directory, and run `build` command to prepare `*.so` and `*.py` files.
-
-```
-python3 setup all
-```
-> Note: If your changes are not loaded, make sure to restart the instance by running `systemctl stop memgraph` and `systemctl start memgraph`.
 
-Next time you change something, just run the following command, since it is we have already set up new directory for
-query modules directory:
+Run the `build` command of the `setup` script. It will generate a `mage/dist` directory with all the `*.so` and `*.py` files.
+Flag `-p (--path)`  represents where will contents of `mage/dist` directory be copied. You need to copy it to
+`/usr/lib/memgraph/query_modules` directory, because that's where Memgraph is looking for query modules by
+[default](https://docs.memgraph.com/memgraph/reference-guide/configuration/).
 
 ```
-python3 setup build
+python3 setup build -p /usr/lib/memgraph/query_modules
```

Comment:
Note that we should include this in Github Action

Reply:
Where?